### ISSUE TITLE
Upgrading to 1.1.0 causes fatal error during build

### ISSUE THREAD

**Issue by mikaelkaron** [ionic-team/stencil-sass#23]
After upgrading to `1.1.0` running `npm start` results in:

```
[ ERROR ]  Cannot find module 'chokidar' Require stack: -
           C:\project\node_modules\@stencil\sass\dist\index.js -
           C:\project\stencil.config.ts -
           C:\project\node_modules\@stencil\core\dist\sys\node\index.js -
           C:\project\node_modules\@stencil\core\bin\stencil Error: Cannot find        
           module 'chokidar' Require stack: -
           C:\project\node_modules\@stencil\sass\dist\index.js -
           C:\project\stencil.config.ts -
           C:\project\node_modules\@stencil\core\dist\sys\node\index.js -
           C:\project\node_modules\@stencil\core\bin\stencil at
           Function.Module._resolveFilename (internal/modules/cjs/loader.js:625:15) at Function.Module._load
           (internal/modules/cjs/loader.js:527:27) at Module.require (internal/modules/cjs/loader.js:683:19) at require      
           (internal/modules/cjs/helpers.js:16:16) at
           C:\project\node_modules\@stencil\sass\dist\index.js:7:1013 at
           Object.<anonymous>
           (C:\project\node_modules\@stencil\sass\dist\index.js:7:1291217) at
           Module._compile (internal/modules/cjs/loader.js:776:30) at Object.Module._extensions..js
           (internal/modules/cjs/loader.js:787:10) at Module.load (internal/modules/cjs/loader.js:643:32) at
           Function.Module._load (internal/modules/cjs/loader.js:556:12)
```

`stencil.config.ts`

```
import { Config } from '@stencil/core';
import { sass } from '@stencil/sass';

// https://stenciljs.com/docs/config

export const config: Config = {
  globalStyle: 'src/global/app.scss',
  globalScript: 'src/global/app.ts',
  plugins: [
    sass({
      includePaths: ['node_modules'],
      injectGlobalPaths: ['src/global/variables.scss', 'src/global/mixins.scss']
    })
  ],
  outputTargets: [
    {
      type: 'www',
      // comment the following line to disable service workers in production
      serviceWorker: null
    },
    {
      type: 'docs-readme'
    }
  ]
};
```

Running with `@stencil/core@1.7.0`

**Comment by jdnichollsc** [comment]
Same error with a new project:
```
[ ERROR ]  Cannot find module 'chokidar' Error: Cannot find module 'chokidar' at
           Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15) at
           Function.Module._load (internal/modules/cjs/loader.js:562:25) at Module.require
           (internal/modules/cjs/loader.js:692:17) at require (internal/modules/cjs/helpers.js:25:18) at
           /Users/jdnichollsc/dev/stenciljs/webrtc-audio/node_modules/@stencil/sass/dist/index.js:7:1013 at
           Object.<anonymous>
           (/Users/jdnichollsc/dev/stenciljs/webrtc-audio/node_modules/@stencil/sass/dist/index.js:7:1291217)
           at Module._compile (internal/modules/cjs/loader.js:778:30) at Object.Module._extensions..js
           (internal/modules/cjs/loader.js:789:10) at Module.load (internal/modules/cjs/loader.js:653:32)
           at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
```

- "@stencil/core": "^1.7.1"
- "@stencil/sass": "^1.1.0"

**Comment by edgarordonez** [comment]
Hey!

Same here when I tried to upgrade [d3-stencil](https://d3-stencil.dev) 👀

**Comment by ozee** [comment]
Ditto, I am also encountering the same issue when using `1.1.0`.

**Comment by warapitiya** [comment]
Temporary solution is to do

```
$ yarn install chokidar
// or
$ npm install chokidar
```

**Comment by itstueben** [comment]
Same Error with new project.
    "@stencil/core": "1.7.3",
    "@stencil/sass": "^1.1.0"

**Comment by bitflower** [comment]
Got it too

Slack reference: https://stencil-worldwide.slack.com/archives/C7987GCA2/p1571932985117100

**Comment by audacitus** [comment]
> ## Temporary Solutions
> ### Solution 1
> ```
> $ yarn install chokidar -D
> // or
> $ npm install chokidar --save-dev
> ```
> 
> ### Solution 2
> ```
> $ yarn add @stencil/sass@1.0.1 -D
> // or
> $ npm install @stencil/sass@1.0.1 --save-dev
> ```

Had a the same issue. sass@1.0.1 worked.

**Comment by flawyte** [comment]
Same issue for me with :
- `@stencil/core` 1.7.4
- `@stencil/sass` 1.1.0

Issue opened 17 days ago, and still no sign of any Stencil team member... Stencil is a great tool but the lack of support on GitHub issues really is a problem.

**Comment by flawyte** [comment]
Same issue with :
- `@stencil/core` 1.7.4
- `@stencil/sass` 1.1.0

Issue opened 17 days ago, and still no sign of any Stencil team member... Stencil is a great tool but the lack of support on GitHub issues really is a problem.

**Comment by adamdbradley** [comment]
Should be fixed as of `1.1.1`. Thanks
